pkg/chunkenc/interface.go

```diff
@@ -55,13 +56,13 @@ func (e Encoding) String() string {
 	case EncDumb:
 		return "dumb"
 	case EncLZ4_64k:
-		return "lz4"
+		return "lz4-64k"
 	case EncLZ4_256k:
 		return "lz4-256k"
 	case EncLZ4_1M:
 		return "lz4-1M"
 	case EncLZ4_4M:
```

Comment:
why 4M the default now ? because the bigger the block is the better the chance are that the compression is better at the expense of memory usage. However using 64k block will create multiple block per chunk block causing a bit more cpu overhead. 

tldr; trading memory for cpu.